spec/ref_arch_setup/install_spec.rb

```diff
@@ -22,29 +22,57 @@
       @expected_command << " --nodes #{target_master}"
     end
 
-    it "got a pass from bolt" do
-      expected_output = "All Good"
-      expected_status = 0
-      expect(install).to receive(:`).with(@expected_command).and_return(expected_output)
-      `(exit #{expected_status})`
-      expect($?).to receive(:success?).and_return(true) # rubocop:disable Style/SpecialGlobalVars
-      expect(install).to receive(:puts).with("Running: #{@expected_command}")
-      expect(install).to receive(:puts).with("Exit status was: #{expected_status}")
-      expect(install).to receive(:puts).with("Output was: #{expected_output}")
-      expect(install.bootstrap_mono(pe_conf_path, pe_tarball_path, target_master)).to eq(true)
+    context "called using default value" do
```

Comment:
Better Specs recommends starting contexts with "when" or "with": http://www.betterspecs.org/#contexts

Not a big deal, but I think this makes it flow more naturally when reading. Otherwise this looks great!